README.md

```diff
@@ -103,6 +107,9 @@ see [usage documentation](https://github.com/sclorg/postgresql-container/tree/ge
 For information about usage of Dockerfile for PostgreSQL 13,
 see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/13).
 
+For information about usage of Dockerfile for PostgreSQL 15,
+see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/15).
```

Comment:
The reference can be directly to https://github.com/sclorg/postgresql-container/tree/master/15

Reply:
Will fix.